pasta_eln/backend.py

```diff
@@ -507,7 +507,7 @@ def useExtractors(self, filePath:Path, shasum:str, doc:dict[str,Any]) -> None:
             except:
               doc[meta][item] = str(doc[meta][item])
               print('**Warning -> stringified  ',meta, item)
-        if doc['-type'][0] in [doc['recipe'].split('/')[0], '-']:
+        if doc['-type'][0] in (doc['recipe'].split('/')[0]):
```

Comment:
_**(doc['recipe'].split('/')[0])**_ will always be a single element right? Since you do the indexing [0]..

If so, why can't we use equality check instead of 'in' operator?

Reply:
@jmurugan-fzj  THANKS, Yes, that was correct, I even made it shorter
- if doc['recipe'].startswith(doc['-type'][0]):